task/01-strings-tasks.js

```diff
@@ -69,7 +69,8 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+    let value2 = value.replace("Hello, ",'')
+    return value2.replace("!",'')
```

Comment:
redundant variable declarations, you just can do `return value.replace(...).replace(...)`